### DISCUSSION TITLE
useSuspenseInfiniteQuery invalidateQueries and refetchQueries pageParam is not reset to initialPageParam

### DISCUSSION THREAD

**Discussion by jamezening** [TanStack/query#7739] [Q&A]
Hi,

I have tried invalidating and refetching a `useSuspenseInfiniteQuery` via `invalidateQueries` and `refetchQueries`, but when doing so, the useSuspenseInfiniteQuery `pageParam` is not being reset (query is being called with pageParam of last page).

rq versions tried
---
"@tanstack/react-query": "^5.14.1",
and
"@tanstack/react-query": "^5.51.1",

I have tried invalidating and refetching a non suspense infinite query, and the query is being called again with pageParam reset as `initialPageParam`.

Has anyone else run into a similar issue?

**Comment by TkDodo** [comment]
please show a minimal reproduction
